src/os/ui/menu/mapmenu.js

```diff
@@ -29,50 +39,51 @@ os.ui.menu.map.setup = function() {
     children: [{
       label: 'Map',
       type: os.ui.menu.MenuItemType.GROUP,
+      sort: os.ui.menu.map.GroupSort.GROUPS++,
```

Comment:
This works nicely here but forces plugins to use decimal values to insert something between them.

Reply:
Fair point. I suppose it also makes the order for plugin menu items dependent on plugin load order... also bad. I'll switch back to the hard coded values and use fixed values for the group sort as well.